### ISSUE TITLE
Consider supporting only OSI-approved licenses

### ISSUE THREAD

**Issue by sdruskat** [citation-file-format/citation-file-format#528]
Reasons:

- The list is much shorter and thus more maintainable

## Potential catch

- OSI may use wrong/different SPDX identifiers

**Comment by hainesr** [comment]
My strong personal preference here is to remain with SPDX. Narrowing the options available to people feels like it would be a reason for some to avoid using CFF. I do understand the maintainability question though.

**Comment by hainesr** [comment]
Yes, I would imagine that going with the intersection of SPDX/OSI is more work than just plain SPDX!

Honestly, I would stick with being as permissive as possible, within some constraints, and sticking with the SPDX superset.

**Comment by rettinghaus** [comment]
[OSI Approved Licenses](https://opensource.org/licenses) only apply to software and are not suited for datasets in general. Especially the CC licenses are missing there, so I would say that restricting the list would lock out a larger set of CFF users.

**Comment by sdruskat** [comment]
Thanks @hainesr and @rettinghaus, let's stick with supporting the full SPDX list then.
